Lib/idlelib/debugger.py

```diff
@@ -130,80 +154,187 @@ def make_gui(self):
         pyshell = self.pyshell
         self.flist = pyshell.flist
         self.root = root = pyshell.root
+        self.clickable_cursor = 'hand2'
+        windowingsystem = self.root.tk.call('tk', 'windowingsystem')
+        if windowingsystem == 'aqua':
```

Comment:
```suggestion
        if self.root._windowingsystem == 'aqua':
```
While this accesses a protected member, I believe this is neater (opinions?). **EDIT:** `windowingsystem` is also used later so maybe this is better.

Reply:
_windowingsystem is private, but not  protected.  It is a property in tkinter.Misc.  When accessed, _windowingsystem_cached is added to avoid repeating the tk.call.  It the time significant?  It is used a couple of places in tkinter for 'x11' specific code for clipboard and selection retrieval.  The private name could theoretically be renamed or even deleted, but I think either unlikely.  I might request that this be made public.  Mark, do what you want here.

In the long run, I would like to add an idle dict/Namespace with attributes like root, windowsys, pointerhand, (all fixed at startup) and zoomheight (set when needed).

Reply:
I used @E-Paine's suggestion... @terryjreedy I agree it would be very convenient if tkinter more visibly exposed this; also agree about a central place for what are effectively application-level globals

Reply:
Resolved